### PR TITLE
Added bee ascii

### PR DESCRIPTION
**Pull request by Eknir**
- Added a simple ascii Bee (source: https://www.asciiart.eu/animals/insects/bees), to be printed to the command line before anything else starts.
- Added a small welcome message

### CODE REVIEW

File: cmd/bee/cmd/start.go [hunk 1]
```diff
@@ -71,6 +71,22 @@ func (c *command) initStartCmd() (err error) {
 			default:
 				return fmt.Errorf("unknown verbosity level %q", v)
 			}
+			bee := `
+Welcome to the Swarm.... Bzzz Bzzzz Bzzzz
```
Comment (vandot) [inline cmd/bee/cmd/start.go:75]: @Eknir there is `B` with 3 z and then two times 4 z this is intentional?

Comment (vandot) [inline cmd/bee/cmd/start.go:75]: @Eknir there is `B` with 3 z and then two times 4 z this is intentional?

**Comment by vandot** [comment]
> @vandot @svetomir would this additional information that is printed to stdout influence how we parse log messages?

@janos we will have couple of lines that could not be parsed but no issues because of thar

**Comment by janos** [comment]
@vandot cool, thanks.

**Pull request merged**